src/System.IO.Compression.ZipFile/pkg/System.IO.Compression.ZipFile.pkgproj

```diff
@@ -2,18 +2,36 @@
 <Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
   <ItemGroup>
+    <ProjectReference Include="..\ref\4.0.0\System.IO.Compression.ZipFile.depproj">
```

Comment:
If we were to make this change you don't need to fork the ref, just change the NETStandard version it targets.